dm/worker/subtask.go

```diff
@@ -30,16 +30,18 @@ import (
 	"github.com/pingcap/dm/syncer"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/failpoint"
 	"github.com/siddontang/go/sync2"
 	"go.uber.org/zap"
-	// hack for glide update, remove it later
-	_ "github.com/pingcap/tidb-tools/pkg/check"
-	_ "github.com/pingcap/tidb-tools/pkg/dbutil"
-	_ "github.com/pingcap/tidb-tools/pkg/utils"
 )
 
 // createUnits creates process units base on task mode
 func createUnits(cfg *config.SubTaskConfig) []unit.Unit {
+	failpoint.Inject("mockWorkerUnits", func(_ failpoint.Value) {
```

Comment:
generalize this failpoint？

Reply:
maybe need to generalize or use a better name because only _dump_ unit used in L42.